tests/Resources/ti.ui.ios.tabbedbar.addontest.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe.ios('Titanium.UI.iOS', function () {
+
+	it('#createTabbedBar()', function () {
+		should(Ti.UI.iOS.createTabbedBar).not.be.undefined;
+		should(Ti.UI.iOS.createTabbedBar).be.a.Function;
+	});
+});
+
+describe.ios('Titanium.UI.iOS.TabbedBar', function () {
+
+	it('#labels', function () {
+		var tabbedBar = Ti.UI.iOS.createTabbedBar({
+			labels: [ 'One', 'Two', 'Three' ],
+		});
+		should(tabbedBar.labels).be.an.Array;
+		should(tabbedBar.getLabels).be.a.Function;
+		should(tabbedBar.labels.length).be.eql(3);
+		should(tabbedBar.getLabels().length).eql(3);
+		tabbedBar.labels = [ 'Four', 'Five'];
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-default;
  File: tests/Resources/ti.ui.ios.tabbedbar.addontest.js;
  Line: 31;
-->

- :warning: tests/Resources/ti.ui.ios.tabbedbar.addontest.js line 31 – A space is required before ']'. (array-bracket-spacing)